pkg/web_css/lib/src/_list.scss

```diff
@@ -80,6 +80,55 @@
   }
 }
 
+.modal-carousel {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  left: 0px;
+  top: 0px;
+  z-index: 1;
+  display: none;
+  background: rgba(0,0,0,0.85);
+}
+
+.carousel-next {
+  right: 10px;
+}
+
+.carousel-prev {
+  left: 10px;
+}
+
+.carousel-nav {
+  background-color: rgba(0,0,0,0.7);
+
+  &:hover {
+    background-color: #4285f4;
+  }
+  position: fixed;
+  top: 50%;
+  transform: translateY(-50%);
+  display: none;
+  z-index: 2;
+}
+
+.carousel_image {
```

Comment:
`carousel-image` instead for consistency?

Reply:
done.